### ISSUE TITLE
Administration button: focused elements are black dark color

### ISSUE THREAD

**Issue by raphj** [tracim/tracim#3956]
## Description and expectations

When the primary color of an instance is set to a dark color (which it should - brown by default), links in the administration button's menu are black on dark background when focused. The text color should be white instead.

### Version information

- Tracim version: v3.3
